bson/bson.go

```diff
@@ -99,12 +99,12 @@ type Getter interface {
 	GetBSON() (interface{}, error)
 }
 
-// A value implementing the bson.Setter interface will receive the BSON
+// Setter interface : a value implementing the bson.Setter interface will receive the BSON
```

Comment:
The space between "interface :" should be removed for consistency with the Getter comment. Very nitpicky this.